exercises/concept/secrets/secrets_test.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+import pytest
+from secrets import secret_add, secret_multiply, secret_max, secret_sort, secret_combine
+
+
+class TestLambdas(unittest.TestCase):
+    @pytest.mark.task(taskno=1)
+    def test_add_3(self):
+        self.assertEqual(secret_add(3)(3), 6)

```

Comment:
All of our test cases should have a human-friendly `msg` parameter for return in the UI.  Please see [this example](https://github.com/exercism/python/blob/main/exercises/concept/guidos-gorgeous-lasagna/lasagna_test.py) from the Guidos Gorgeous Lasagna exercise.